src/script/components/asset/fileAsset.js

```diff
@@ -120,7 +106,7 @@ ko.components.register('file-asset', {
   `,
   viewModel: {
     createViewModel(params, component_info) {
-      return new z.components.FileAssetComponent(params, component_info);
+      return new FileAssetComponent(params, component_info);
```

Comment:
Maybe rename `component_info` to `componentInfo`? Not a must, but a nice-to-have.